src/libaktualizr/crypto/keymanager.cc

```diff
@@ -162,7 +162,7 @@ std::string KeyManager::getCa() const {
 }
 
 std::string KeyManager::getCN() const {
-  std::string not_found_cert_message = "Certificate is not found, can't extract device_id";
+  const std::string not_found_cert_message = "Certificate is not found, can't extract device_id";
```

Comment:
True, but this is actually kind of tedious to do since it could be in a file, in an SQL database, or in an HSM. Besides, this function is called once during initialization, and a failure here would imply something seriously wrong with the configuration or filesystem.